tests/server/api/api_test.go

```diff
@@ -1094,11 +1094,26 @@ func TestPreparingProgress(t *testing.T) {
 
 func sendRequest(re *require.Assertions, url string, method string, statusCode int) []byte {
 	req, _ := http.NewRequest(method, url, http.NoBody)
-	resp, err := tests.TestDialClient.Do(req)
-	re.NoError(err)
+	var (
+		err  error
+		resp *http.Response
+	)
+	testutil.Eventually(re, func() bool {
+		//nolint:bodyclose
```

Comment:
why ignore this lint? you can place `defer resp.Body.Close() ` into this function.

Reply:
+1

Reply:
It is closed in L1108 already